SEP-0002-revised.md

```diff
@@ -0,0 +1,244 @@
+
+* Current commit access to the core repo, or any accepted affiliated package maintained by the project, and with a commit to that repository within the last 12 months.
+* Currently holds a role listed under the community roles section of the website: https://sunpy.org/project/#community-roles.
+* Is a currently serving board member.
+
+### Adding New Project Members
+
+Anyone is eligible to be nominated to be a Project member, as long as they meet the criteria laid out in the Membership section above.
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the steering committee.
+The steering committee, and the Ombudsperson check that the nomination is factually accurate and their record of contribution to the community adheres to the Code of Conduct.
+1. The candidate is then put to the Project Members.
+The name and nomination statement of the candidate will be shared with the Project Members.
+The candidate can become a Project member if five existing Project members approve the candidate and no Project Members call for an election within two weeks of the candidate being put to the Project Members.
+1. Project Members can call for an election either by posting in the discussion of the candidate or reaching out privately to the Steering Committee or the Ombudsperson.
+If a vote is called for then the candidate will be come a Project Member with a simple majority of all votes cast in the election.
+An election needs to be announced to the Project Members one week before the voting opens and voting will be open for two weeks.
+1. The candidate will be informed promptly of the outcome by the Steering Committee.
+
+The name of a candidate should not be shared outside of the Project Members until they become a Project Member.
+A public list of Project Members will be kept on the SunPy website.
+
+Nominations, discussions and any elections of Project Members should take place before the initial election of the Steering Committee.
+
+### Active and Emeritus Status
+
+Project Members have no term limits.
+Project Members who are no longer actively contributing to the project should contact the Steering Committee to move themselves to emeritus status.
+Active Project Members who have not made any significant contributions for two years, will be contacted by the Steering Committee and be asked to move themselves to emeritus status.
+The Steering Committee can change a Project Member's status to emeritus.
+Emeritus Project Members will continue to be listed on the website.
+Emeritus Project Members are not able to participate in votes.
+An emeritus member may request to be marked as active at any time if they feel they are making contributions.
+
+## Removing Project Members
+
+In exceptional circumstances, it may be necessary to remove someone from the Project Members against their will.
+A vote must be held to remove a Project Member.
```

Comment:
```suggestion
A vote of the project membership must be held to remove a Project Member.
```
Is my suggestion correct? Or is it another group who vote?

Reply:
That is correct.